src/components/forms/SubnetTileForm.js

```diff
@@ -65,21 +65,52 @@ class SubnetTileForm extends React.Component {
           className="marginBottomSmall"
         />
         <div className="displayFlex">
-          {subnetMap.map((subnet) => (
-            <SubnetForm // change so doesn't show buttons
-              key={`${subnet.name}-tile-${this.props.tier}-${
-                this.props.vpc_name
-              }-${JSON.stringify(subnet)}`}
-              vpc_name={this.props.vpc_name}
-              data={subnet}
-              onSave={this.props.onSave}
-              isModal={this.props.isModal || this.props.readOnly}
-              componentDidUpdateCallback={this.childSubnetHasChanged}
-              networkAcls={this.props.networkAcls}
-              disableSaveCallback={this.props.disableSaveCallback}
-              shouldDisableGatewayToggle={this.shouldDisableGatewayToggle}
-            />
-          ))}
+          {subnetMap.map((subnet, index) => {
```

Comment:
does this have Alex's fix? it looks like subnets is still indented

<img width="500" alt="Screenshot 2023-04-26 at 12 22 22 PM" src="https://user-images.githubusercontent.com/11401353/234681565-e080304a-8280-4286-b2a4-5ff63df0eb66.png">

Reply:
it should? i pulled main before i started but i'll check

Reply:
yes it does, not sure what's up here